JuiceMaker/JuiceMaker/Controller/FruitQuantityViewController.swift

```diff
@@ -10,7 +10,93 @@ import UIKit
 final class FruitQuantityViewController: UIViewController {
     static let id = String(describing: FruitQuantityViewController.self)
     
+    @IBOutlet weak var kiwiLabel: UILabel!
+    @IBOutlet weak var mangoLabel: UILabel!
+    @IBOutlet weak var pineappleLabel: UILabel!
+    @IBOutlet weak var bananaLabel: UILabel!
+    @IBOutlet weak var strawberryLabel: UILabel!
+    
+    @IBOutlet weak var strawberryStepper: UIStepper!
+    @IBOutlet weak var bananaStepper: UIStepper!
+    @IBOutlet weak var pineappleStepper: UIStepper!
+    @IBOutlet weak var kiwiStepper: UIStepper!
+    @IBOutlet weak var mangoStepper: UIStepper!
+    
+    private let juiceMaker = JuiceMaker()
+    private let fruitStore = FruitStore.shared
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        hideBackButton()
+        updateStockToLabel()
+    }
+    
+    private func hideBackButton() {
+        navigationItem.hidesBackButton = true
+    }
+    
+    private func updateStockToLabel() {
+        fruitStore.fruitQuantity.forEach{(key, value) in
+            switch key {
+            case .strawberry:
+                strawberryLabel.text = String(value)
+            case .banana:
+                bananaLabel.text = String(value)
+            case .kiwi:
+                kiwiLabel.text = String(value)
+            case .mango:
+                mangoLabel.text = String(value)
+            case .pineapple:
+                pineappleLabel.text = String(value)
+            }
+        }
+    }
+    
+    private func safelyUnwrap(fruit: Fruit) -> Int {
+        guard let fruit = fruitStore.fruitQuantity[fruit] else { return 0 }
+        return fruit
+    }
+    
+    private func transformStringToInt(label: String?) -> Int {
+        guard let stringLabel = label, let number = Int(stringLabel) else { return 0 }
+        return number
+    }
```

Comment:
```suggestion
    private func transformStringToInt(label: String?) -> Int {
        guard let stringLabel = label,
                   let number = Int(stringLabel) else { return 0 }
        return number
    }
```
한 줄에 코드가 다 존재한다면 가독성이 떨어지게 됩니다.